runtime/doc/lsp.txt

```diff
@@ -0,0 +1,359 @@
+*lsp.txt* The Language Server Protocol
+
+        NVIM REFERENCE MANUAL
+
+
+Neovim Language Server Protocol (LSP) API
+
+Neovim exposes a powerful API that conforms to Microsoft's published Language
+Server Protocol specification. The documentation can be found here:
+
+	https://microsoft.github.io/language-server-protocol/
+
+
+================================================================================
+                                                            *lsp-server-config*
+
+Neovim only exposes a powerful API for the language server protocol. To get
```

Comment:
Sounds weird  to say that "neovim  *only* exposes a powerful API" (if it is powerful, why be modest? ;)) Suggestion: "Neovim exposes a powerful API..., but to get...".